### PR TITLE
Changes for change about us label for browser language. 

### PR DESCRIPTION
**Pull request by development0261**
Changes for change about us label for browser language.

### CODE REVIEW

**Comment by cbleek** [comment]
this is not the way i18n should work.

please take a look at the example:

https://medium.com/quasar-framework/adding-full-i18n-to-quasar-150da2d5bba4

in Step 5 there is e working [codesandbox example](https://codesandbox.io/s/0b5n9)

**Comment by Hetal2425** [comment]
Hi @cbleek ,
Ok I will check it.

**Comment by Hetal2425** [comment]
Hi @cbleek ,
I did commit code from fork Hetal2425 / cross-solution.de.
I did not get proper language using  **$ i18n.locale**
That is why I have used  **$ q.lang.getLocale ()** to get the browser language.
Please see this video https://www.screencast.com/t/lxRzg0c7aii for **$ i18n.locale** Problems.

File: src/layouts/CrossLayout.vue [hunk 1]
```diff
@@ -201,7 +201,10 @@ export default {
   },
   mounted () {
     if (this.$q.lang.getLocale().indexOf('en') === 0) {
-      this.label = 'About Us'
+      import(`quasar/lang/${this.$q.lang.getLocale()}`).then(language => {
+        this.$q.lang.getLocale()(language.default)
+      })
+      this.label = this.$t('aboutus')
```
Comment (cbleek) [inline src/layouts/CrossLayout.vue]: we are getting closer to the point. I'll merge this. But I still not understand, why you are using a variable.

**Comment by cbleek** [comment]
@Hetal2425 your branch is a couple commits behind. Can you pull the latest commits to bring your changes to the HEAD.

Auswahl_403

**Comment by Hetal2425** [comment]
Hi @cbleek ,

I have started to working on the task.

Facing some to display About Us without variable.

I will look the issues by tomorrow.

**Comment by Hetal2425** [comment]
Hi @cbleek  ,
Merry Christmas, May this year bring lots of joy and happiness in your life and family. 🙂
Please check I have solved About-us label issues without using variable.

**Comment by cbleek** [comment]
It's now working as expected.

there are still warnings. We'll fix them in a new branch

**Comment by rajeshp0261** [comment]
Okay @cbleek .
Any new task for me?

**Pull request merged**